Makefile

```diff
@@ -83,9 +83,10 @@ test/coverage:
 	go tool cover -html=filtered_cover.out -o coverage.html
 	rm cover.out filtered_cover.out
 
-.PHONY: test-sanity
-test-sanity:
-	go test -v -race ./tests/sanity/...
+# TODO: Re-enable sanity tests
```

Comment:
np: Can we add to this comment to explain why sanity tests are disabled, and what we need to do in order to re-enable them?

Reply:
Added comment